includes/utils.hpp

```diff
@@ -31,6 +34,9 @@ std::string trimWhiteSpaces(const std::string &str);
 
 void splitByCommaAndTrim(const std::string &argument, stringVector *args);
 
+KeyValuePairList parseCommandIntoKeyValuePairList(const std::string &key,
+                                                  const std::string &value);
+
```

Comment:
It should be a static function in `join.hpp` because it's not shared function but only used for `join`.

Reply:
in fact i wanted to use the same funciton for join and mode that is why i put it in utils. 

but i now rename parseCommandIntoKeyValuePairList to parsejoin